model-server-test/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+
+plugins {
+    kotlin("jvm")
+    alias(libs.plugins.docker.compose)
+}
+
+dependencies {
+    testImplementation(kotlin("test"))
+    testImplementation(project(":model-server"))
+}
+
+tasks.test {
+    doFirst {
+        val db = dockerCompose.servicesInfos.getValue("db")
+        systemProperty("jdbc.url", "jdbc:postgresql://${db.host}:${db.port}/")
+    }
+}
+
+dockerCompose {
+    isRequiredBy(tasks.test)
+    setProjectName("model-server-test-started-from-gradle")
+
+    // Works around an issue where the docker command is not found.
+    // Falls back to the default, which may work on some platforms.
+    // https://github.com/avast/gradle-docker-compose-plugin/issues/435
+    // https://github.com/gradle/gradle/issues/10483
+    if (DefaultNativePlatform.getCurrentOperatingSystem().isMacOsX) {
+        listOf("/usr/bin/docker-compose", "/usr/local/bin/docker-compose").firstOrNull {
+            File(it).exists()
+        }?.let { docker ->
+            executable.set(docker)
+        }
+        listOf("/usr/bin/docker", "/usr/local/bin/docker").firstOrNull {
+            File(it).exists()
+        }?.let { docker ->
+            dockerExecutable.set(docker)
+        }
+    }
```

Comment:
What about a gradle property that users can use to give their non-standard location of `docker-compose` if `PATH` is not an option?